conda/rapids-tpcx-bb.yml

```diff
@@ -27,13 +25,11 @@ dependencies:
   - psutil
   - ipykernel
   - jupyterlab
-  - torchvision
-  - pytorch
   - gspread
   - oauth2client
   - pip
   - pip:
     - transformers
```

Comment:
We should also remove this.

Reply:
good catch

Reply:
Resolved by https://github.com/rapidsai/gpu-bdb/pull/139/commits/a180bc797fe0d9304ca7628f9e41bc47544f3034